ts/parts/Series.ts

```diff
@@ -218,6 +219,10 @@ declare global {
         interface Chart {
             runTrackerClick?: boolean;
         }
+        interface DataExtremesObject {
+            dataMin?: number;
+            dataMax?: number;
```

Comment:
Why are the properties optional? Also consider use of `RangeObject` interface with `min` / `max`.

Reply:
In some cases, like when boosting is active, or in general for the windbarb series, there are no data extremes. So I thought it was better to return an object with undefined keys than to return undefined.

> Also consider use of RangeObject interface with min / max.

Yes, we could use that. Then we would have to change the above to check for existence of the whole object. Any preferences @pawelfus , @bre1470 ?

Reply:
I'm fine with all solutions (return: undefined, object without keys or object with keys that values are undefined):

- `undefined` is the easiest to work with (instead of checking `isNumber(dataExtremes.dataMax)` check if `dataExtremes` are defined), but at the same time it's a little strange that `getExtremes()` returns nothing
- `{ dataMin: undefined|number, dataMax: undefined|number }` is kind of standarized output from this function I would expect, assuming that dataMin/Max can be undefined eg when data is empty

Reply:
Yes I agree to the second point (or both actually). But the second point is the reason why I did it this way, given that `series.dataMin` can also be undefined.